app/templates/components/currency-amount.hbs

```diff
@@ -0,0 +1 @@
+{{#if (gt @amount 0)}} {{@currency}} {{/if}}{{@amount}}
```

Comment:
There is no space between currency and amount

Reply:
its there